Enable/Models/Location.h

```diff
@@ -13,5 +13,6 @@
 @property (nonatomic, strong) PFGeoPoint * _Nonnull coordinates;
 @property (nonatomic, strong) NSString * _Nonnull name;
 @property (nonatomic, strong) NSString * _Nonnull address;
+@property long reviews;
```

Comment:
I assume this is not a list of reviews but an average rating right? Might want to use a better name.

Also are you sure you need a long here? I feel like you probably need a float in order to have decimal numbers, if integer is fine, do you really need a long rather than an int?